packages/toolpad-app/src/toolpad/AppEditor/PageEditor/ComponentPanel.tsx

```diff
@@ -97,6 +101,13 @@ export default function ComponentPanel({ className }: ComponentPanelProps) {
             )}
           </TabPanel>
           <TabPanel value="theme" className={classes.panel}>
+            <Typography className={classes.themesDocsLink} variant="body2">
+              Customize the app with a MUI theme. Read more about this in the{' '}
+              <Link href="https://mui.com/toolpad/concepts/theming" target="_blank" rel="noopener">
```

Comment:
This is a 301, slowing down the page load speed.

```suggestion
              <Link href="https://mui.com/toolpad/concepts/theming/" target="_blank" rel="noopener">
```

Fixed in ddffda5885f97c1ed7ac87dfe2a822c0e8672309